frontend/src/scenes/surveys/SurveySettings.tsx

```diff
@@ -64,21 +111,26 @@ export function SurveySettings(): JSX.Element {
         })
     }
 
+    if (isModal) {
+        return <SurveyPopupToggle />
+    }
+
     return (
-        <>
-            <div className="flex items-center gap-2 mb-2">
-                <LemonField.Pure className="mt-2" label="Appearance">
-                    <span>These settings apply to new surveys in this organization.</span>
-                </LemonField.Pure>
+        <div className="flex flex-col gap-2">
+            <SurveyPopupToggle />
+            <LemonDivider className="m-0" />
 
-                <div className="flex-1" />
+            <div className="flex items-center gap-1 flex-1 justify-between">
+                <LemonField.Pure label="Appearance" className="text-base gap-1">
+                    <span className="text-sm">These settings apply to new surveys in this organization.</span>
```

Comment:
Just a question: If I have those settings, create a new survey, and set the appearance configuration, which one will be the precedent?

Reply:
this is only for new surveys, really. it's set like this: `{ ...defaultSurveyAppearances, ...appearanceDefinedInTheSettings }`

after that, what you write on the survey form directly is what matters, as appearance for each survey is stored individually